packages/checkout/widgets-lib/src/locales/en.json

```diff
@@ -328,22 +328,26 @@
         "WALLET_REJECTED": {
           "description": "You'll need to approve the transaction in your wallet to proceed.",
           "primaryAction": "Try again",
-          "secondaryAction": "Cancel"
```

Comment:
shouldn't this be updated in the other langs too?

Reply:
thanks, this is unwanted rolling back